src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-form/iscsi-target-form.component.html

```diff
@@ -11,12 +11,10 @@
       <div class="card-body">
         <!-- Target IQN -->
         <div class="form-group row">
-          <label class="col-form-label col-sm-3"
-                 for="target_iqn">
-            <ng-container i18n>Target IQN</ng-container>
-            <span class="required"></span>
-          </label>
-          <div class="col-sm-9">
+          <label class="cd-col-form-label required"
```

Comment:
Does this also need to be applied to e.g. https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-discovery-modal/iscsi-target-discovery-modal.component.html#L13 ?

Reply:
Done.